.pipelines/aks-addon-nightly/e2e-aks-addon-jobs.yaml

```diff
@@ -1,6 +1,6 @@
 jobs:
   - job: test_osm_aks_addon
-    pool: staging-pool
```

Comment:
why is this changing?

Reply:
Old staging pools / ADO agents are obsolete - Upstream Publishing is required to use the Mariner agents now.